src/main/flight/pid.c

```diff
@@ -86,13 +87,33 @@ void pidResetErrorAngle(void)
 
 void pidResetErrorGyro(void)
 {
-    errorGyroI[ROLL] = 0;
-    errorGyroI[PITCH] = 0;
-    errorGyroI[YAW] = 0;
+    int axis;
+    for (axis = 0; axis < 3; axis++) {
+        errorGyroI[axis] = 0;
+        errorGyroIf[axis] = 0.0f;
+    }
+}
+
+float scaleItermToRcInput(int axis) {
+    float rcCommandReflection = (float)rcCommand[axis] / 500.0f;
+    static float iTermScaler[3] = {1.0f, 1.0f, 1.0f};
+    static float antiWindUpIncrement = 0;
+
+    if (antiWindUpIncrement) antiWindUpIncrement = 0.001 * (targetLooptime / 500);  // Calculate increment for 500ms period
```

Comment:
We need the result in floats

Looptime 3000
(1 / 500) * 3000 = 6

0.001 * (3000 / 500) = 0,006